tests/Client/TestUpdater.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tuf\Tests\Client;
+
+use Tuf\Client\RepoFileFetcherInterface;
+use Tuf\Client\Updater;
+use Tuf\Tests\TestHelpers\TestClock;
+
+/**
+ * An updater class for testing using the \Tuf\Tests\TestHelpers\TestClock class.
+ */
+class TestUpdater extends Updater
+{
+    public function __construct(RepoFileFetcherInterface $repoFileFetcher, array $mirrors, \ArrayAccess $durableStorage)
+    {
+        parent::__construct($repoFileFetcher, $mirrors, $durableStorage);
+        $this->clock = new TestClock();
```

Comment:
@tedbow You need to be able to inject a mock of the clock in order to control its behavior. You probably want to add it to the constructor arguments.

Reply:
Yep. thanks fixed.

Reply:
I'm not seeing where this is fixed...?

Reply:
@phenaproxima `\Tuf\Tests\Client\TestUpdater::__construct()` now has a `$testClock` argument

Reply:
So in the future we could provide `$testClock` that have the time where root is not expired but another file like snapshot.json is.